.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -3,21 +3,31 @@ name: New release template
 about: Template for creating new releases of Durable Functions
 title: ''
 labels: ''
-assignees: cgillum
+assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
 ---
 
-As part of a release, the following items need to be taken care of. For any items that are not applicable, they should be crossed out using the `~~ ~~` markdown syntax with an explanation.
-
-- [ ] Run private scale tests and ensure no regressions
-- [ ] Publish updated versions of [DurableTask.Core](https://www.nuget.org/packages/Microsoft.Azure.DurableTask.Core/) and [DurableTask.AzureStorage](https://www.nuget.org/packages/Microsoft.Azure.DurableTask.AzureStorage/) to nuget.org
+**Prep Release: (assigned to:)**
+_Due: <two-businessdays-before-release>_
 - [ ] Merge all features and fixes into the `master` branch
-- [ ] Update .NET API docs at https://azure.github.io/azure-functions-durable-extension
-- [ ] Publish signed version of [Microsoft.Azure.WebJobs.Extensions.DurableTask](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.DurableTask/) to nuget.org
-- [ ] Update samples to point to the latest nuget packages
+- [ ] Publish updated versions of DurableTask.Core and DurableTask.AzureStorage to app-service MyGet feed
+- [ ] Increment extension version
+- [ ] Publish signed version of Microsoft.Azure.WebJobs.Extensions.DurableTask to app-service MyGet feed
+
+**Validation (assigned to: )**
+_Due: <1-businessdays-before-release>_
+- [ ] Run private performance tests and ensure no regressions
+- [ ] Smoke test Functions V1 and Functions V3 .NET apps
+- [ ] Smoke test JavaScript and Python apps
+
+
+** Release Completion (assigned to: )**:
+_Due: <release-deadline>_
+- [ ] Push staged package on MyGet to Nuget.org (for Durable Task Framework, you may need to manually update them)
 - [ ] Create a PR in the [Azure Functions templates repo](https://github.com/Azure/azure-functions-templates) to update templates to the latest version
 - [ ] Create a PR in the [Azure Functions bundles repo](https://github.com/Azure/azure-functions-extension-bundles) to update bundles to the latest version
 - [ ] Close out or punt remaining GitHub issues for the current milestone
-- [ ] Update official docs under https://docs.microsoft.com/en-us/azure/azure-functions/durable ([private docs repo for Microsoft employees](http://github.com/MicrosoftDocs/azure-docs-pr))
-- [ ] Publish release notes
+- [ ] Merge all pending PR docs, and reset pending_docs.md
+- [ ] Publish release notes and reset release_notes.md
+- [ ] Remerge to master now that pending docs and release notes are cleared?
```

Comment:
There seems to be an accidental leftover `?` here, right?

```suggestion
- [ ] Remerge to master now that pending docs and release notes are cleared
```

Reply:
The question mark is because I'm not sure if this extra step is necessary or desirable.

Reply:
Fair enough. Maybe it could read "If necessary, re-merge to master to ensure pending docs and release docs are included?"
I don't feel strongly though, just a thought.

Reply:
I reordered steps and reassigned them so this second merge isn't necessary.